products/motorola.md

```diff
@@ -0,0 +1,429 @@
+---
+title: Motorola Mobility
+category: device
+tags: mobile-phone
+iconSlug: motorola
+permalink: /motorola
+alternate_urls:
+-   /motorolamobility
+-   /motorola-mobility
+releasePolicyLink: https://en-us.support.motorola.com/app/software-security-update
+releaseColumn: false
+releaseDateColumn: true
+activeSupportColumn: false
+eolColumn: Security Updates
+
+# This data is parsed from the HTML CDATA on the Motorola website:
+# https://en-us.support.motorola.com/app/software-security-update
+#
+# See this Gist for a parsing script:
+#     https://gist.github.com/ab/67c6343530c6b8e599f07cedccbcba3d
+#
+
+releases:
+- releaseCycle: "moto g stylus 5G (2023)"
+  releaseDate: 2023-06-01
+  support: true
```

Comment:
```suggestion
```

The `support` attribute is not needed when `activeSupportColumn=false`.

Reply:
Yeah I was waffling on whether to display that column. In the source script, it outputs `support: true` when the upstream Motorola page says `Currently receiving regular security updates` for a given product.

But that column conflicts with the `eol` column, because there are several products whose EOL dates have passed, where the Motorola website still says "Currently receiving regular security updates." (e.g., moto e20.)

So I hid the column for now, assuming that the EOL dates are more accurate, but I don't know for sure what's actually true.

Reply:
Yes, I understand. Given it's difficult to know whether or not there are actual updates for those models I still think we should stick to what's documented on the support site. Thanks for removing those.